ddtrace/contrib/django/patch.py

```diff
@@ -6,6 +6,10 @@
 `django.apps.registry.Apps.populate` is patched to add instrumentation for any
 specific Django apps like Django Rest Framework (DRF).
 """
+try:
+    from collections.abc import Iterable
```

Comment:
We use `typing.Iterable` in a few integrations but `collections.Iterable` here. We should try to be consistent. We can probably move lines 9 to 12 to `ddtrace/internal/compat.py` and import Iterable from this module.

Reply:
Fixed